webpack.config.js

```diff
@@ -137,7 +137,11 @@ module.exports = [
 
             'flacEncodeWorker':
                 './react/features/local-recording/'
-                    + 'recording/flac/flacEncodeWorker.js'
+                    + 'recording/flac/flacEncodeWorker.js',
+            'analytics-ga':
```

Comment:
Maybe a dumb question but... Why don't we bundle these? How much would the bundle size increate? Other analytics handlers can be synamically loaded, but why do these need to be separate bundles?

Reply:
Why would we? Since we anyway have a generic way of loading them. Is there any benefit?